app/assets/stylesheets/nfg_ui/network_for_good/core/nfg_theme/_sizing.scss

```diff
@@ -0,0 +1,11 @@
+// Width and height based on viewport
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-up($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
+    @each $prop, $abbrev in (width: w, height: h) {
+      @each $size, $length in $sizes {
+        .#{$abbrev}#{$infix}-#{$size} { #{$prop}: $length !important; }
```

Comment:
@kylejmorin if we're indicating our breakpoint support classes in display-app we should update the docs to include the new breakpoint sizing.

Reply:
We have the breakpoints documented here: https://zeroheight.com/213vpgf/p/44ea6c-layout